blazor/chart/axis-customization.md

```diff
@@ -375,6 +375,106 @@ To place an axis in the opposite position of its original position, set its [Opp
 
 ![Blazor Column Chart Axis in Opposed Position](images/axis-customization/blazor-column-chart-axis-at-opposed-position.png)
 
+## Scroll Bar Positioning
```

Comment:
This topic must be placed in Zooming page. See EJ2 https://helpej2.syncfusion.com/documentation/chart/zooming#position